__tests__/app.test.js

```diff
@@ -0,0 +1,22 @@
+const request = require('supertest');
+
+const app = require('../app.js');
+
+const seed = require('../db/seeds/seed.js');
+const testData = require('../db/data/test-data/');
+const db = require('../db/connection.js');
+
+describe('app', () => {
+
+  beforeEach(() => seed(testData));
+
+  afterAll(() => db.end());
+
+  describe('/api/topics', () => {
+    describe('GET', () => {
+      it('200: responses with topics array each containing slug and description properties', () => {
+        
+      })
+    })
+  })
```

Comment:
In a separate describe block(for errors maybe) I would recommend testing for you 404 error. You will also need to ensure this error is dealt with in and error file (along with your 505) I can see in app.js you have referenced an errors file but I cannot see this file or it's content on your PR. Please add this error handling and submit another PR :)